### PR TITLE
Fix Plaid modal by updating to latest version of `react-native-plaid-link-sdk` / resolve stashed token issue

### PR DESCRIPTION
**Pull request by marcaaron**
### Details
cc @Julesssss @luacmartins 

### Fixed Issues

$ https://github.com/Expensify/App/issues/4959

### Tests AND QA Steps
1. Start with a non-public domain account that has not yet set up a workspace or bank account
2. Create a new workspace by tapping on the green FAB button
3. Navigate to `/bank-account` by selecting the workspace and tapping the "Get Started" button
4. Tap button to add account via Plaid (Note: This step may take time to complete)
5. Enter test credentials `user_good` `pass_good`
6. Press "Continue"
7. Verify that the account selection dropdown appears

### Tested On

- [ ] Web
- [ ] Mobile Web
- [ ] Desktop
- [x] iOS
- [x] Android

### CODE REVIEW

File: src/CONFIG.js [hunk 1]
```diff
@@ -60,6 +60,6 @@ export default {
         DEFAULT: '/favicon.png',
         UNREAD: '/favicon-unread.png',
     },
-    CAPTURE_METRICS: lodashGet(Config, 'CAPTURE_METRICS', false),
-    ONYX_METRICS: lodashGet(Config, 'ONYX_METRICS', false),
+    CAPTURE_METRICS: lodashGet(Config, 'CAPTURE_METRICS', 'false') === 'true',
+    ONYX_METRICS: lodashGet(Config, 'ONYX_METRICS', 'false') === 'true',
```
Comment (marcaaron) [inline src/CONFIG.js:64]: **Note:** These changes are not related, but I noticed my test builds were showing the metrics and decided to fix this here.

**Comment by marcaaron** [comment]
This should be ready for review now

**Comment by marcaaron** [comment]
So... _actually_.... I'm still experiencing issues on Android on debug builds. Building for release works fine. @yuwenmemon lmk if you need help figuring out how to do this (there is a SO post somewhere).

**Comment by marcaaron** [comment]
Hmm actually it's worse than I thought and also happens in release but inconsistently. 
Gonna put this back into draft and file see if I can get some help from Plaid.

**Comment by marcaaron** [comment]
Ok, this is ready for a review now.

There is still an issue with Plaid that causes the UI to load very slowly and basically makes it seem broken on dev, but I got to the bottom of the worst of the Android issues. Plaid is aware of the issue and working on it - it's related to their web bundle that is getting served via a WebView. But not sure what we can do about this from our end at the moment.

A more extreme idea is to create our own native module, but Plaid is also working on that for `react-native` so it seems like we should `#do-nothing` and wait for improvements.

**Review by luacmartins — COMMENTED** [review]
Thanks for working on this @marcaaron! The Plaid modal is opening on Android now! I still see the following error when confirming the account though. Is that part of the issue that Plaid is trying to fix?

```
Invariant Violation: No callback found with cbID 17868 and callID 8934 for  PlaidAndroid.startLinkActivityForResult - most likely the callback was already invoked. Args: '[{"publicToken":"public-sandbox-53eb79a5-bfdd-4aa5-80c7-e45fa0695ed0","metadata":{"metadataJson":"{\"institution\":{\"name\":\"Chase\",\"institution_id\":\"ins_3\"},\"account\":{\"id\":null,\"name\":null,\"type\":nul...(truncated)...","linkSessionId":"0f3901e3-83f7-48b3-8191-e12c818c3247","institution":{"name":"Chase","id":"ins_3"},"accounts":[{"subtype":"checking","type":"depository","mask":"0000","name":"Plaid Checking","id":"jq958bm9a6s5Errllw8AtW3pPd35lWFy9ez3n"},{"subtype":"savings","type":"depository","mask":"1111","name":"Plaid Saving","id":"7Mxb9p5xeNFZ3zzjjDwEhxvDRVv1gxCN3qAGv"}]}}]', js engine: hermes
```

**Comment by luacmartins** [comment]
I'll try testing again tomorrow!

**Review by Julesssss — APPROVED** [review]
Working flawlessly for me on Android devices, thanks for looking into this. Personally, I've never found the slow loading time to be that problematic, but nice to hear Plaid is looking into improvements anyway.

**Comment by luacmartins** [comment]
Just tested again and it's working!

**Review by luacmartins — APPROVED** [review]

**Comment by marcaaron** [comment]
>Personally, I've never found the slow loading time to be that problematic, but nice to hear Plaid is looking into improvements anyway.

Oh nice! It could be related to my device or internet now that I think about it. Glad to hear it's only slow for me!

**Comment by marcaaron** [comment]
@Julesssss @luacmartins thanks for testing! 🙇

**Comment by isagoico** [comment]
@marcaaron @luacmartins Hello! We're currently unable to test this in iOS and Android since we can't navigate to a specific URL in-app This might have to be tested internally or if you know another method please let us know.

**Comment by marcaaron** [comment]
Same answer as here -> https://github.com/Expensify/App/pull/5029#issuecomment-912664141 but please let us know here if that's not going to work! Thanks!

**Comment by isagoico** [comment]
This issue is failing this PR https://github.com/Expensify/App/issues/5075

**Pull request merged**